From: Ludovic Rousseau <ludovic.rousseau@free.fr>
Date: Tue, 19 Dec 2017 16:02:52 +0100
Subject: Add documentation to the systemd file

Fixes lintian warning:
I: pcscd: systemd-service-file-missing-documentation-key lib/systemd/system/pcscd.service
---
 etc/pcscd.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/etc/pcscd.service.in b/etc/pcscd.service.in
index 7934050..eb093de 100644
--- a/etc/pcscd.service.in
+++ b/etc/pcscd.service.in
@@ -1,6 +1,7 @@
 [Unit]
 Description=PC/SC Smart Card Daemon
 Requires=pcscd.socket
+Documentation=man:pcscd(8)
 
 [Service]
 ExecStart=@sbindir_exp@/pcscd --foreground --auto-exit
